core/tools.js

```diff
@@ -123,6 +123,42 @@
 	 * @singleton
 	 */
 	CKEDITOR.tools = {
+
+		/**
+		 * Verifies the safety of an html string and returns trusted version.
```

Comment:
another nit about the comment: we don't technically verify the safety, since it's an "reviewed conversion" it just means it's deemed safe by a reviewer.